processor/src/main/resources/org/mapstruct/ap/internal/model/source/builtin/JaxbElemToValue.ftl

```diff
@@ -19,7 +19,10 @@
      limitations under the License.
 
 -->
-private <T> T ${name}( JAXBElement<T> element ) {
+<#assign jaxbElement = findType("JAXBElement")>
+<#assign typeElement = jaxbElement.typeParameters.get(0)>
+
+private <<@includeModel object=typeElement/>> <@includeModel object=typeElement/> ${name}( <@includeModel object=jaxbElement/> element ) {
```

Comment:
This method is actually a generic method with it looks like:

```
private <T> T something( JAXBElement<T> element)
```

which is correct. In order to get what you are looking for there is a property that you can pass. The template would then look like:

```
private <T> T ${name}( <@includeModel object=findType("JAXBElement") raw=true /> ) {
```

The `Type.ftl` doesn't use the generic parameters then